app/serializers/property_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class PropertySerializer < ActiveModel::Serializer
+  attribute :compact_domains
```

Comment:
this is break change and it may affect system unpredictable ways. we need to list all fields we have now at property to avoid possible issues (e.g. edit property dialog isn't working on map upload page) and keep backward compatibility.